ui/admin/app/components/form/host-catalog/aws/index.hbs

```diff
@@ -398,7 +398,7 @@
       name='disable_credential_rotation'
       checked={{@model.disable_credential_rotation}}
       disabled={{form.disabled}}
-      {{on 'change' (fn this.toggleDisableCredentialRotation @model)}}
+      {{on 'change' (fn @toggleDisableCredentialRotation @model)}}
```

Comment:
I see you've pulled this up to the shared component but how come this didn't have to change for azure as well?

Reply:
I didn't make any changes to `azure` because it is a disabled field and users can't make any changes